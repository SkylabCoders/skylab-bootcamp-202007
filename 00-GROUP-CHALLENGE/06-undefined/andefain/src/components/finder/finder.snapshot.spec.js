import renderer from 'react-test-renderer';
import React from 'react';
import Finder from './finder';
//import { BrowserRouter as Router } from 'react-router-dom';

describe('Finder snapshot', () => {
	const finderTree = renderer.create(<Finder />);
	it('Should match', () => {
		expect(finderTree.toJSON()).toMatchSnapshot();
	});
});
