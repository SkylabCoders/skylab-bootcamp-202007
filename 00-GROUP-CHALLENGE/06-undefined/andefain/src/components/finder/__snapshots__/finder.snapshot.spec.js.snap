// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Finder snapshot Should match 1`] = `
<section
  className="finder-container"
>
  <div
    className="finder-no-results"
  >
    No results found
  </div>
</section>
`;
